docs/releasing.md

```diff
@@ -1,11 +1,15 @@
-# Mithril Release Processes
+<!--meta-description
+Describes how we do releases of Mithril.js
+-->
+
+# Mithril.js Release Processes
 
 **Note** These steps all assume that `MithrilJS/mithril.js` is a git remote named `mithriljs`, adjust accordingly if that doesn't match your setup.
 
-- [Releasing a new Mithril version](#releasing-a-new-mithril-version)
+- [Releasing a new Mithril.js version](#releasing-a-new-mithril-version)
```

Comment:
Does the anchor still work after this change? See below.

Reply:
It won't. https://github.com/MithrilJS/mithril.js/pull/2316#discussion_r810236379